Ruch_silnikami.py

```
@@ -519,23 +599,32 @@ def move_forward(self, distance):
         target_position = abs(distance)  # target travel distance in cm
 
-        self.ncoder_floor.reset_cunter()
+        self.ncoder_floor.reset_counter()
         current_position = 0.0
         self.is_moving = True
 
         try:
-            while current_position < target_position:
+            while current_position <= target_position:
+
+                if(self.endstop_floor_1 == True):
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if self.endstop_floor_1:` for truth checks

Replace with `self.endstop_floor_1`

(E712)